logging/logger.go

```diff
@@ -14,6 +16,10 @@ type Logger interface {
 	Sublogger(subname string) Logger
 	AddAppender(appender Appender)
 	AsZap() *zap.SugaredLogger
+
+	CDebug(ctx context.Context, args ...interface{})
```

Comment:
[q] Why are we not adding `C` versions of other methods? Because context versions of log methods are only needed for context-level debug sensitivity?

Reply:
Valid question, I waffled on this. I think we need to add it for the other methods too. Let's discuss when you're in the office next week.

Reply:
Sounds good! Up to you if you want to add the `CDebug`s now and the `CInfo`s and `CErrors`s later (could be good to add them all at once to only break the interface one time).

Reply:
I actually did that work a couple weeks ago (one at a time as you suggested). But the [PR structure](https://github.com/viamrobotics/rdk/pull/3286) was the same.

Reply:
Gotcha.. yeah I saw that other PR (and I think a third one) associated with the ticket.